Jenkinsfile

```
@@ -81,25 +89,30 @@ void build_taichi() {
     $CC --version
     $CXX --version
     echo $WORKSPACE
+    . "/home/buildbot/miniconda3/etc/profile.d/conda.sh"
+    conda activate $CONDA_ENV
     $PYTHON_EXECUTABLE -m pip install --user setuptools astor pybind11 pylint sourceinspect
     $PYTHON_EXECUTABLE -m pip install --user pytest pytest-rerunfailures pytest-xdist yapf
-    $PYTHON_EXECUTABLE -m pip install --user numpy GitPython coverage colorama autograd
-    export TAICHI_REPO_DIR=$WORKSPACE/
+    $PYTHON_EXECUTABLE -m pip install --user numpy GitPython coverage colorama autograd torch
```

```suggestion
    $PYTHON_EXECUTABLE -m pip install --user numpy GitPython coverage colorama autograd
```

I believe `torch` is installed properly.